src/main/resources/i18n/messages.properties

```diff
@@ -65,6 +65,7 @@ user.view.create.label=Please, provide your information
 user.view.create.title=Create User
 user.firstname=Name
 user.lastname=Last Name
+user.phone=Phone Number
 user.email=Email
```

Comment:
Currently we support EN and ES languages, therefore we need to add `user.phone` in this file: 
https://github.com/josdem/vetlog-spring-boot/blob/main/src/main/resources/i18n/messages_es.properties
Translation should be: Número Telefónico

Reply:
@josdem 
I tried adding it to the Spanish file, but please check if there is a problem mm

Reply:
Looks good now 👍